Polyfills/Window/Source/Window.cpp

```diff
@@ -88,10 +135,8 @@ namespace
         {
             {
                 std::unique_lock<std::mutex> lk(m_mutex);
-                while (!m_queue.empty())
-                {
-                    m_queue.pop();
-                }
+                m_idMap.clear();
+                m_timeMap.clear();
                 m_shutdown = true;
```

Comment:
`m_shutdown` isn't protected inside the mutex lock in the thread proc. It is probably okay on most platforms, but maybe we should use `std::atomic<bool>` or `arcana::cancellation` instead.